remappings.txt

```diff
@@ -0,0 +1,9 @@
+forge-std=lib/forge-std/src
+@yieldnest-vault=lib/yieldnest-flex-strategy/lib/yieldnest-vault/src
+lib/openzeppelin-contracts=lib/yieldnest-flex-strategy/lib/yieldnest-vault/lib/openzeppelin-contracts
+lib/openzeppelin-contracts-upgradeable=lib/yieldnest-flex-strategy/lib/yieldnest-vault/lib/openzeppelin-contracts-upgradeable
+openzeppelin-contracts-upgradeable/=lib/yieldnest-flex-strategy/lib/yieldnest-vault/lib/openzeppelin-contracts-upgradeable
+openzeppelin-contracts/=lib/yieldnest-flex-strategy/lib/yieldnest-vault/lib/openzeppelin-contracts
+lib/yieldnest-vault/=lib/yieldnest-flex-strategy/lib/yieldnest-vault
+script=lib/yieldnest-flex-strategy/script
```

Comment:
this part of remappings.txt is pretty whacky.. 

it ca'nt detect the flex-strategy files in script so i added these 2 lines to solve this.

i hope there's a simpler sollution